src/txdb.h

```diff
@@ -17,6 +17,18 @@
 class CBlockFileInfo;
 class CBlockIndex;
 struct CDiskTxPos;
+
+// START insightexplorer
+struct CAddressUnspentKey;
+struct CAddressUnspentValue;
+struct CAddressIndexKey;
+struct CAddressIndexIteratorKey;
+struct CAddressIndexIteratorHeightKey;
+
+typedef std::pair<CAddressUnspentKey, CAddressUnspentValue> CAddressUnspent;
+typedef std::pair<CAddressIndexKey, CAmount> CAddressIndexAmount;
```

Comment:
How about `CAddressUnspentEntry` and `CAddressIndexEntry`.

Reply:
I think the crucial thing to communicate is the structs represent pairs which I think entry does. I personally wouldn't have included `Db`, but I think that's fine.